spec/features/passenger_management_spec.rb

```diff
@@ -65,7 +65,7 @@
       click_link 'Add New Passenger'
       fill_in('passenger[name]', with: 'Foo Bar')
       fill_in('passenger[email]', with: 'foobar@invalid.com')
-      fill_in('doctors_note_expiration_date', with: date)
+      fill_in('passenger[doctors_note_attributes][expiration_date]', with: date)
```

Comment:
It would be good, perhaps in a separate PR, to fix the form so that the view tests can work properly. Example: https://github.com/umts/st-pax/pull/81/files